files/build_templates/init_cfg.json.j2

```diff
@@ -16,5 +16,21 @@
             "{{crm_res}}_high_threshold": "85"{% if not loop.last %},{% endif -%}
 {% endfor %}
         }
+    },
+    "Feature": {
+{%- for feature in ["sflow", "telemetry"] %}
+        "{{feature}}": {
+            "status": "disabled"
+        }{% if not loop.last %},{% endif -%}
+{% endfor %}
+    },
+    "CONTAINER_FEATURE": {
+{%- for container in ["teamd", "swss", "syncd", "pmon", "lldp", "bgp", "dhcp-relay",
+                      "radv", "sflow", "snmp", "telemetry", "database"] %}
```

Comment:
I prefer if you kept these in alphabetical order, as in my example.

Reply:
Changed the order according to alphabetical.